src/demos/calcite-shell.html

```diff
@@ -37,567 +44,56 @@ <h2>Demo Navigation</h2>
           <li>View the <a href="calcite-shell-full-window.html" target="_blank">Full window demo</a>.</li>
```

Comment:
I believe this link needs to be updated.

Reply:
I ended up removing these in most pages (missed this one) since we have them on the main nav under 'advanced'. Lmk if you see value in keeping some inline links to the other shell demos.